engine/gamesys/src/gamesys/components/comp_sprite.cpp

```diff
@@ -65,6 +66,31 @@ namespace dmGameSystem
         SpriteResourceOverrides() : m_Material(0) {}
     };
 
+    const uint32_t MAX_TEXTURE_COUNT = dmRender::RenderObject::MAX_TEXTURE_COUNT;
+    const uint8_t CACHE_EVICTION_FRAMES = 10; // how many frames cache entry is stored before eviction
+
+    struct AnimationData
+    {
+        dmDoubleLinkedList::ListNode    node;
+        // Used after resolving info from all textures
+        dmhash_t                        m_AnimationID;                      // The animation of the driving atlas
+        uint32_t                        m_Frames[MAX_TEXTURE_COUNT];        // The resolved frame indices
+        uint32_t                        m_LastAccessTick;
```

Comment:
Use engine tick as a marker for cache eviction.
I can use for ticks counting `uint16_t` instead of `uint32_t` because overflow case correctly handles.